testing/auth.ts

```diff
@@ -130,6 +133,36 @@ class CredentialHandler {
     print('Credentials updated!');
   }
 
+  handleCustom(
+    paramDefs: Array<{
```

Comment:
This can be `CustomAuthParameter[]` right?

Reply:
good catch